src/translators/engines/libretranslate.ts

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios'
+import TranslateEngine, { TranslateOptions, TranslateResult } from './base'
+import { Config } from '~/core'
+
+export default class LibreTranslate extends TranslateEngine {
+  apiRoot = 'http://localhost:5000'
```

Comment:
It seems they have the online API, shouldn't we default to it?

Reply:
really do you have a link ?

Reply:
as mentioned here https://github.com/LibreTranslate/LibreTranslate/issues/106 they prefer not to use the public site. as we can't have "no default" I think the current one is the best (default when you run libretranslate locally)